apps/backend/core/configuration/validation/orchestrator.py

```
@@ -0,0 +1,450 @@
+
+    # In production, raise error on critical failures
+    if fail_fast and report.has_critical_failures:
+        error_msg = f"Startup validation failed: {report.to_summary()}"
+        logger.error(error_msg)
+        raise RuntimeError(error_msg)
+
+    return report
+
+
+async def validate_service(
+    config: Any,
```

Ruff (0.14.4)

Dynamically typed expressions (typing.Any) are disallowed in `config`

(ANN401)